core/tracing.go

```diff
@@ -30,7 +30,8 @@ func StartDutyTrace(ctx context.Context, duty Duty, spanName string, opts ...tra
 	ctx, outerSpan = tracer.Start(tracer.RootedCtx(ctx, traceID), fmt.Sprintf("core/duty.%s", strings.Title(duty.Type.String())))
 	ctx, innerSpan = tracer.Start(ctx, spanName, opts...)
 
-	outerSpan.SetAttributes(attribute.Int64("slot", safeInt64(duty.Slot)))
+	slotStr := strconv.FormatUint(duty.Slot, 10)
+	outerSpan.SetAttributes(attribute.String("slot", slotStr))
```

Comment:
@xenowits @dB2510 this line changes the "slot" tracing attribute from int64 to string.

I don't think it'll lead to any worrisome behavior, but since you guys have the most experience on the codebase, wdyt?

Reply:
I don't think it should cause any ripple behaviors